japanese/beanus/beanus.md.txt

```diff
@@ -0,0 +1,4 @@
+﻿川の音を頼りに、森の中にぐんぐん分け入ってと、綺麗な泉を見つけたあなた。
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.